src/DGtal/doc/tutorials/moduleShortcuts.dox

```diff
@@ -430,7 +430,7 @@ Shortcuts::Point is Z3i::KSpace::Point.
   - Shortcuts::getSurfelRange: returns the surfels of a digital surface in the specified traversal order.
   - Shortcuts::getIdxSurfelRange: returns the indexed surfels of an indexed digital surface in the specified traversal order.
   - Shortcuts::getPointelRange: returns the pointels of a digital surface in the default order and optionnaly the map Pointel -> Index giving the indices of each pointel, or simply the pointels around a surfel.
-  - Shortcuts::saveOBJ: saves a digital surface as an OBJ file, with optionally positions, normals and colors information.
+  - Shortcuts::saveOBJ: saves a digital surface as an OBJ file, with optionally positions, normals and colors information. You may also add a vector-field to your OBJ file.
```

Comment:
I don't get it.. You can provide normals AND an additional vector field ?
Isn't the second one supposed to be normal as well ?

Reply:
clarified in the doc